config/jobs/kubernetes/sig-k8s-infra/trusted/sig-security-trusted.yaml

```diff
@@ -11,79 +11,21 @@ periodics:
   decorate: true
   extra_refs:
   - org: kubernetes
-    repo: kubernetes
-    base_ref: master
-    path_alias: k8s.io/kubernetes
+    repo: sig-security
+    base_ref: main
+    workdir: true
   spec:
+    serviceAccountName: k8s-snyk-scan
     containers:
     - image: golang
       envFrom:
       - secretRef:
           # secret key should be defined as SNYK_TOKEN
           name: snyk-token
```

Comment:
... we should really mount this as a file and pass it into snyk that way instead of an env

Do we really need this to run in the trusted cluster anyhow though? It doesn't seem like this account is all that sensitive, but the other things in that cluster are and I'd rather have less arbitrary code running there.

(Alternatively, we could run this in a cloudbuild like the image pushing jobs and sandbox it that way)